pcdsdevices/attenuator.py

```diff
@@ -1093,6 +1202,10 @@ class AT2L0(FltMvInterface, PVPositionerPC, LightpathInOutMixin):
     blade_18 = Cpt(FEESolidAttenuatorBlade, ':MMS:18')
     blade_19 = Cpt(FEESolidAttenuatorBlade, ':MMS:19')
 
+    def summarize_err(self):
+        """returns the error summary """
```

Comment:
This docstring is no longer true, the error summary is printed rather than returned.